src/manager.cpp

```diff
@@ -46,9 +47,74 @@ sdbusplus::async::task<> Manager::init()
         co_await startFullSync();
     }
 
+    _ctx.spawn(monitorSignalForDebug());
     co_return co_await startSyncEvents();
 }
 
+// NOLINTNEXTLINE
+sdbusplus::async::task<> Manager::monitorSignalForDebug()
+{
+    while (!_ctx.stop_requested())
+    {
+        co_await sdbusplus::async::sleep_for(_ctx, std::chrono::seconds(1));
+        std::signal(SIGTRAP, logSignalHandler);
```

Comment:
Other services use SIGUSR1 to dump data.

Also, why do you need to install a signal handler more than once?